README.md

```diff
@@ -43,7 +43,9 @@ We strive for collaboration with [mutual respect for each other][contributing].
 
 ### Discussion
 
-We're all on Mozilla's IRC in the [#devtools-html][irc-devtools-html] channel on irc.mozilla.org.
+Our primary communication method is now on the [Slack app][slack]
+
+We are also on Mozilla's IRC in the [#devtools-html][irc-devtools-html] channel on irc.mozilla.org.
 
```

Comment:
I wonder if we can simplify the language a bit and make it consistent with the other options.

```
* **Chat** Say hello in [Slack][slack] or in the [#devtools-html][irc-devtools-html] channel on irc.mozilla.org. 
```

@clarkbw?